scripts/globals/items/bunch_of_fortune_fruits.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------------
+-- ID: 6498
+-- Item: Bunch of Fortune Fruits
+-- Food Effect: 30Min, All Races
+-----------------------------------------
+-- Charisma +7
+-- may have unknown hidden effects
+-----------------------------------------
+require("scripts/globals/status")
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0
+    if target:hasStatusEffect(dsp.effect.FOOD) or target:hasStatusEffect(dsp.effect.FIELD_SUPPORT_FOOD) then
+        result = 246
```

Comment:
GET YOUR MAGIC NUMBERS OUT OF HERE

`dsp.msg.basic.IS_FULL`

Reply:
:scream: how did that get in here!

Reply:
and if nudging gets it reviewed, I'll keep nudging. else I might never notice I dun goofed.